runtime/parachains/src/router/ump.rs

```diff
@@ -96,6 +111,20 @@ impl<T: Trait> Module<T> {
 
 		// make sure that the queue is not overfilled.
 		// we do it here only once since returning false invalidates the whole relay-chain block.
+		if para_queue_count > config.max_upward_queue_count {
+			log::warn!(
+				target: LOG_TARGET,
+				"para_queue has more items than permitted by config ({} > {})",
```

Comment:
```suggestion
				"the ump queue has more items than permitted by config ({} > {})",
```

Reply:
also, it's not yet happened, so I guess it should be "would have" instead of "hash".